### DISCUSSION TITLE
My C drive keeps filling up!

### DISCUSSION THREAD

**Discussion by hoodady** [lllyasviel/stable-diffusion-webui-forge#626] [Q&A]
I have forge in Z drive, and while using forge on Chrome, for some reason, it keeps filling my C drive to the point that I go out of storage. This is particularly after 5 - 6 generations on SDXL or Controlnet....My available memory keeps dropping.. not sure whats the issue and hoping someone can help please...
![image](https://github.com/lllyasviel/stable-diffusion-webui-forge/assets/76712106/0fadb58b-8bdd-4906-9ab0-4a294dc6b922)

**Comment by ScreNN** [comment]
> I have forge in Z drive, and while using forge on Chrome, for some reason, it keeps filling my C drive to the point that I go out of storage. This is particularly after 5 - 6 generations on SDXL or Controlnet....My available memory keeps dropping.. not sure whats the issue and hoping someone can help please...

What kind of video card do you have? How is your Windows swap file configured?

**Reply by KevinCou** [reply]
me pasa lo mismo tengo una nvidia 3070
como puedo solucionarlo?
